lib/ews/types/item.rb

```diff
@@ -356,6 +356,7 @@ def build_modified_occurrences(occurrences)
           elems = a[:occurrence][:elems]
 
           h[DateTime.parse(elems.find{|e| e[:original_start]}[:original_start][:text])] = {
+            item_id: elems.find{|e| e[:item_id]}[:item_id][:attribs][:id],
```

Comment:
Add item_id to the response hash, when calling 'modified_occurrences' on a recurring event. Currently, there is no existing calls to this method, and I am adding information to the response, so I consider this to be a safe, non-breaking change.